crates/ef-testing/src/evm_sequencer/utils.rs

```diff
@@ -82,57 +82,54 @@ pub fn to_broadcasted_starknet_transaction(
-                Felt::ONE,                          // call_array_len
-                *KAKAROT_ADDRESS.0.key(),           // CallArray to
-                selector!("eth_send_transaction"),  // CallArray selector
-                Felt::ZERO,                         // CallArray data_offset
-                calldata.len().into(),              // CallArray data_len
-                calldata.len().into(),              // calldata_len
-            ];
-            execute_from_outside_calldata.append(&mut calldata);
-            execute_from_outside_calldata.push(signature.len().into());
-            execute_from_outside_calldata.append(&mut signature);
-
-            let mut execute_entrypoint_calldata = vec![
-                Felt::ONE,                                    // call_array_len
-                starknet_address,                             // CallArray to
-                selector!("execute_from_outside"),            // CallArray selector
-                Felt::ZERO,                                   // CallArray data_offset
-                (execute_from_outside_calldata.len()).into(), // CallArraydata data_len
-                (execute_from_outside_calldata.len()).into(), // calldata length
-            ];
-            execute_entrypoint_calldata.append(&mut execute_from_outside_calldata);
-            execute_entrypoint_calldata
-        }
-        #[cfg(feature = "v1")]
-        {
-            let mut execute_calldata = vec![
-                Felt::ONE,                         // call_array_len
-                *KAKAROT_ADDRESS.0.key(),          // CallArray to
-                selector!("eth_send_transaction"), // CallArray selector
-                calldata.len().into(),             // CallArray data_len
-            ];
-            execute_calldata.append(&mut calldata);
-            execute_calldata
-        }
-        #[cfg(not(any(feature = "v0", feature = "v1")))]
-        {
-            vec![]
-        }
-    };
+        let mut execute_from_outside_calldata =
+            if cfg!(feature = "v0") {
+                let mut execute_from_outside_calldata = vec![
+                    *RELAYER_ADDRESS.0.key(),           // OutsideExecution caller
+                    Felt::ZERO,                         // OutsideExecution nonce
+                    Felt::ZERO,                         // OutsideExecution execute_after
+                    Felt::from(10_000_000_000_000u128), // OutsideExecution execute_before
+                    Felt::ONE,                          // call_array_len
+                    *KAKAROT_ADDRESS.0.key(),           // CallArray to
+                    selector!("eth_send_transaction"),  // CallArray selector
+                    Felt::ZERO,                         // CallArray data_offset
+                    calldata.len().into(),              // CallArray data_len
+                    calldata.len().into(),              // calldata_len
```

Comment:
the only diff if this line
Maybe remove the if block and append only for v0 ?

Reply:
I prefer to have it explicit here as it's not super straightforward - in two blocks the transaction structure is outlined better